### ISSUE TITLE
Not implemented: window.alert

### ISSUE THREAD

**Issue by Igor-Vuk** [rstacruz/jsdom-global#25]
After upgrading to the new version of jsdom and jsdom-global I am getting this issue. Code that I am testing has this line inside of it  window.alert('Please try again'). It was not a problem until now?

Thanks

**Comment by DcsMarcRemolt** [comment]
It seems, most of the not DOM specific globals are now gone. Things like window.Object also give me undefined now.

**Comment by Igor-Vuk** [comment]
Yeah. Not sure how to resolve this. I wanna test that alert showed up but always getting the not implemented message.

**Comment by modosc** [comment]
this is actually a jsdom issue, workaround is here:

https://github.com/tmpvar/jsdom/issues/1843

**Comment by jonknowles** [comment]
It seems window.Object and window.Math are also not available, so I also had to roll back to jsdom < 10

**Comment by cowardcalf** [comment]
Any updates on this issue?
